packages-exp/auth-exp/src/core/persistence/persistence_user_manager.ts

```diff
@@ -84,6 +90,23 @@ export class PersistenceUserManager {
     }
   }
 
+  _onStorageEvent(_value: PersistenceValue | null): void {
+    assert(this.listener, AuthErrorCode.INTERNAL_ERROR, { appName: this.auth.name });
+    this.listener();
```

Comment:
this is a change from the original, we only allow one listener now in the user manager (the auth object), and we move all the local storage cruft into local storage persistence implementation

Reply:
makes sense